tests/ocf-url_parse-path-absolute/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
+    <head>
+        <meta charset="utf-8"/>
+        <title>Parse a root-relative URL</title>
```

Comment:
"Root-relative image reference, parsed by JS"?

Or something better than that; I'm not clear on the distinction between this and https://github.com/w3c/epub-tests/blob/main/tests/ocf-url_link-path-absolute/EPUB/package.opf, other than the JS. I think their titles should be as close as possible.

Reply:
done